adal/src/main/java/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -810,6 +811,8 @@ private IWindowComponent wrapActivity(final Activity activity) {
         }
 
         return new IWindowComponent() {
+            //CHECKSTYLE:OFF: checkstyle:MemberName
+            //CHECKSTYLE:OFF: checkstyle:VisibilityModifier
             Activity mRefActivity = activity;
```

Comment:
make mRefActivity private, checkstyle will stop complaining after that (so you do not need the checkstyle:off comments). I was asking if doing this would break anything?